### PR TITLE
[FluidDynamics] Improvement cfd relaxed acceleration variable

### PR DESCRIPTION
**Pull request by sunethwarna**
This PR adds relaxed acceleration to a separate variable in the bossak scheme.

relaxed_acceleration(n) = (1-alpha)*acceleration(n) + alpha*acceleration(n-1)

### CODE REVIEW

File: applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h [hunk 1]
```diff
@@ -285,8 +285,9 @@ namespace Kratos {
 
                     array_1d<double, 3 > & CurrentAcceleration = (itNode)->FastGetSolutionStepValue(ACCELERATION, 0);
                     array_1d<double, 3 > & OldAcceleration = (itNode)->FastGetSolutionStepValue(ACCELERATION, 1);
+                    array_1d<double, 3 > & RelaxedAcceleration = (itNode)->FastGetSolutionStepValue(RELAXED_ACCELERATION, 0);
```
Comment (philbucher) [inline applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h]: Does this have to be a historical variable?
Doesn’t seem so

Comment (sunethwarna) [reply applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h]: Time history of this is required to calculate adjoint sensitivities. At the moment, we are calculating it inside an HDF5 process and storing it in the nodes. This is to move the calculation outside the HDF5 application and to have a cleaner implementation.

Comment (philbucher) [reply applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h]: Ofc course you need the time history, I get that

But you are only accessing step “0“

I will talk to you tmr

**Comment by sunethwarna** [comment]
@jcotela , @philbucher thanks for the comments. I made the changes :) @msandre

**Review by msandre — APPROVED** [review]
I don't have any additional remarks.

**Pull request merged**